app/models/form.rb

```diff
@@ -35,9 +35,10 @@ def self.receivable_association
   scope(:by_name, -> { order("forms.name") })
   scope(:default_order, -> { by_name })
 
-  delegate :arrange_descendants,
-    :children,
+  delegate :children,
+    :preordered_descendants,
```

Comment:
If this is not being used elsewhere in the app then I don't think we should add it to the class' public interface like this. You always want to keep the public interface as small as possible.